fastFM/tests/test_ffm.py

```diff
@@ -38,5 +52,17 @@ def test_ffm2_predict_w0():
     y_pred = ffm2.ffm_predict(w0, w, V, X)
     assert_equal(y_pred, w0)
 
-if __name__ == '__main__':
-    test_ffm2_predict()
+# def test_ffm_fit():
+#     *_, y, X = get_test_problem()
+#     fm = als.FMRegression(n_iter=1000, init_stdev=0.1, rank=2, l2_reg_w=0.1, l2_reg_V=0.5)
+#     w0, w, V = ffm.ffm_als_fit(fm, X, y)
+#     y_pred = ffm.ffm_predict(w0, w, V, X)
+#
+#     assert_equal(np.round(y_pred), y)
+
+def test_ffm2_fit():
+    w0, w, V, y, X = get_another_test_problem()
+    rank = 2
+    w0, w, V = ffm2.ffm_als_fit(w0, w, V, X, y, rank)
+    y_pred = ffm2.ffm_predict(w0, w, V, X)
+    assert_equal(np.round(y_pred), y)
```

Comment:
We could compare msqr before and after fit?
`from sklearn.metrics import mean_squared_error`

Reply:
Aw, that's cool! My fault. With such assert we don't need any changes in core2 settings.

Reply:
Good, we just need to keep in mind that this way of testing is not exhaustive. For example all error reduction could come from one parameter set (for example w0) while the rest is actually getting worse (due to wrong memory mapping etc.). 

Might be good to do this test once with rank=0 and once with rank>0. In principle the fit with rank>0 should be better but this also depends on the data and settings (number of iterations etc.).

Reply:
Yep, but we cannot use different params for easy testing from python now. Maybe we should switch to serialization or any other way to send the solver params to core2 and come back here after that?

Reply:
We could check if all parameters changed their value, this together with the current assert should be good enough for this PR. What do you think?